include/RAJA/pattern/params/reducer.hpp

```diff
@@ -27,13 +27,11 @@ struct ValLoc {
   RAJA_HOST_DEVICE ValLoc(value_type v) : val(v) {}
   RAJA_HOST_DEVICE ValLoc(value_type v, RAJA::Index_type l) : val(v), loc(l) {}
 
-  RAJA_HOST_DEVICE void min(value_type v, index_type l) { if (v <  val) { val = v; loc = l; } }
-  RAJA_HOST_DEVICE void max(value_type v, index_type l) { if (v >  val) { val = v; loc = l; } }
+  RAJA_HOST_DEVICE void min(value_type v, index_type l) { if (v < val) { val = v; loc = l; } }
+  RAJA_HOST_DEVICE void max(value_type v, index_type l) { if (v > val) { val = v; loc = l; } }
 
-  bool constexpr operator < (const ValLoc& rhs) const { return val <= rhs.val; }
-  bool constexpr operator <=(const ValLoc& rhs) const { return val < rhs.val; }
-  bool constexpr operator > (const ValLoc& rhs) const { return val >= rhs.val; }
-  bool constexpr operator >=(const ValLoc& rhs) const { return val > rhs.val; }
+  bool constexpr operator<(const ValLoc& rhs) const { return val < rhs.val; }
+  bool constexpr operator>(const ValLoc& rhs) const { return val > rhs.val; }
```

Comment:
Did we want to keep the `<=` and `>=` operators?

Reply:
I don't know.
I removed them because we don't support == in this class or in ValueLoc (which also doens't have `<=` `>=`). Also we ignore loc in the comparison.
On the other hand they are equivalent and often implemented in terms of each other `a >= b` = `b < a`.

Reply:
That's fair. For the sake of consistency with ValueLoc, it's probably best to leave out the `=` comparators.